WowPacketParserModule.V8_0_1_27101/Parsers/CharacterHandler.cs

```diff
@@ -0,0 +1,206 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.Parsing;
+using WowPacketParser.Store;
+using WowPacketParser.Store.Objects;
+using WowPacketParserModule.V8_0_1_27101.Enums;
+
+namespace WowPacketParserModule.V8_0_1_27101.Parsers
+{
+    public static class CharacterHandler
+    {
+        [Parser(Opcode.SMSG_QUERY_PLAYER_NAME_RESPONSE)]
+        public static void HandleNameQueryResponse(Packet packet)
+        {
+            var hasData = packet.ReadByte("HasData");
+
+            packet.ReadPackedGuid128("Player Guid");
+
+            if (hasData == 0)
+            {
+                packet.ReadBit("IsDeleted");
+                var bits15 = (int)packet.ReadBits(6);
+
+                var count = new int[5];
+                for (var i = 0; i < 5; ++i)
+                    count[i] = (int)packet.ReadBits(7);
+
+                for (var i = 0; i < 5; ++i)
+                    packet.ReadWoWString("Name Declined", count[i], i);
+
+                packet.ReadPackedGuid128("AccountID");
+                packet.ReadPackedGuid128("BnetAccountID");
+                packet.ReadPackedGuid128("Player Guid");
+
+                packet.ReadUInt64("Unk801"); // equals Unk801 in ReadCharactersData (SMSG_ENUM_CHARACTERS_RESULT)
```

Comment:
CommunityDbID
```

ServerToClient: SMSG_QUERY_PLAYER_NAME_RESPONSE (0x2707) Length: 127 ConnIdx: 0 Time: 08/13/2018 22:00:30.172 Number: 7704
HasData: 0
PlayeGuid: Full: 0x081E240000000000000000000973F4BC; HighType: Player; Low: 158594236; Map: 0
IsDeleted: False
[0] Name Declined: Аккари
[1] Name Declined: Аккари
[2] Name Declined: Аккари
[3] Name Declined: Аккари
[4] Name Declined: Аккари
AccountID: Full: 0x740000000000000000000000029362A8; HighType: WowAccount; Low: 43213480; Map: 0
BnetAccountID: Full: 0x78000000000000000000000007920903; HighType: BNetAccount; Low: 127011075; Map: 0
Player Guid: Full: 0x081E240000000000000000000973F4BC; HighType: Player; Low: 158594236; Map: 0
Unk801: 972777519670621372
VirtualRealmAddress: 50790415
Race: 29 (29)
Gender: 1 (Female)
Class: 8 (Mage)
Level: 110
Name: Аккари


ClientToServer: CMSG_QUERY_COMMUNITY_NAME (0x368C) Length: 15 ConnIdx: 2 Time: 08/13/2018 22:00:43.219 Number: 8949
BNetAccountGUID: Full: 0x78000000000000000000000007920903; HighType: BNetAccount; Low: 127011075; Map: 0
CommunityDbID: 972777519670621372

Reply:
afaik you're able to be in multiple clubs, question is which of these it is